pull_into_place/structures.py

```diff
@@ -286,6 +302,19 @@ def read_and_calculate(workspace, pdb_paths):
                 record[key] = np.max(residue_specific_restraint_distances[i])
                 metadata[meta.name] = meta
 
+        if len(residue_specific_restraint_angles) > 1:
+            for i in residue_specific_restraint_angles:
+                res = '{0}{1}'.format(sequence_map[i], i)
+                key = 'restraint_angle_{0}'.format(res.lower())
+                meta = ScoreMetadata(
+                        name=key,
+                        title='Restraint Satisfaction for {0} (Rad)'.format(res),
+                        guide=0.5, lower=0.0, upper='95%', order=3,
+                )
+                record[key] = np.max(residue_specific_restraint_angles[i])
+                metadata[meta.name] = meta
+
+
```

Comment:
I mean, I personally prefer rads, if only because the numbers are more comparable to the restraint distances (vaguely similar cutoffs for what's considered "good"). Idk, I don't care much either way but I wouldn't say degrees are universally better.

Reply:
Having numbers in the 0.5-1.0 range doesn't seem very valuble to me, since it's easy to get the GUI to draw the dashed line at 30° or something as a point of reference.  I still think degrees would be better, but it's not an important point, so if you prefer radians that's fine.